src/mh-core.jl

```diff
@@ -46,7 +46,9 @@ struct MetropolisHastings{D} <: MHSampler
 end
 
 StaticMH(d) = MetropolisHastings(StaticProposal(d))
+StaticMH(d::Int) = MetropolisHastings(StaticProposal(MvNormal(FillArrays.Zeros(d), I)))
```

Comment:
AFAICT is FillArrays neither loaded nor a direct dependency yet. Can you fix this?

Reply:
Done!